buildSrc/src/main/groovy/airbyte-python.gradle

```diff
@@ -25,7 +25,7 @@ class AirbytePythonPlugin implements Plugin<Project> {
             pip 'mypy:0.812'
             pip 'isort:5.6.4'
             pip 'pytest:6.1.2'
-            pip 'pip:21.1'
```

Comment:
Was there a reason to upgrade this?

Reply:
Initially testing to see if upgrading pip would get rid of our errors, since wheels are sometimes available in later version. It did not.

Now I'm keeping it because it works so why not :)